dags/fivetran_acoustic.py

```diff
@@ -135,6 +138,7 @@ def _generate_acoustic_report(conn_id: str, report_type: str, config: Dict[Any,
 TAGS = [Tag.ImpactTier.tier_1]
 
 for report_type, _config in REPORTS_CONFIG.items():
+    # IMPORTANT that BQETL DAG follows the following naming convention: bqetl_acoustic_{report_type}
```

Comment:
Could this be an `assert` or `ValueError`?

Reply:
actually, that's a good point. I'll add that. Thanks!